### ISSUE TITLE
Support for Lumen without Facades?

### ISSUE THREAD

**Issue by jonaholsson** [dwightwatson/validating#132]
Any plans to support Lumen with `$app->withFacades();` disabled in `/bootstrap/app.php` and referencing the libraries manually (e.g. `Input` and `Validator`)?

**Comment by dwightwatson** [comment]
Yeah, wondering what the best approach to access those services would be then. The facades are currently used in the tests. I suppose one approach would be to use `app('Illuminate\Validation\Factory')` and `app('Illuminate\Http\Request')` but not sure how I would update the tests for that.

Happy for any other suggestions or PRs, I'd like to see Lumen support sans-Facades if possible.

**Comment by jonaholsson** [comment]
I'll take a look and see what can be done, but won't have time until end of next week I'm afraid.
